lib/publiccloud/gcp_client.pm

```diff
@@ -71,7 +72,27 @@ sub get_next_vault_role {
 sub create_credentials_file {
     my ($self) = @_;
     my $credentials_file;
+    if (get_var('PUBLIC_CLOUD_CREDENTIALS_URL')) {
+        my $json = get_credentials();
+        # Need to remove quotes, convert \n to \\n and / to \/
+        assert_script_run(q{export key=$(cat /root/credentials.json | jq '.private_key_data' | sed 's/\\"//g' | sed 's/\\\n/\\\\\\\n/g' | sed 's/\\//\\\\\//g')});
+        #assert_script_run(q{export key=$(cat /root/credentials.json | jq '.private_key_data' | sed 's/\"//g' | sed 's/\\n/\\\\n/g'      | sed 's/\//\\\//g')});
```

Comment:
Leftover?

Reply:
yes, but debug info until the PR is ready. The `\` are not escaped in this line.